### ISSUE TITLE
Please provide the example for maven-enforcer-plugin to update build status

### ISSUE THREAD

**Issue by jdoriyastpl** [trivago/cucable-plugin#48]

**Comment by jdoriyastpl** [comment]
@laxersaz  Can you please provide the complete example for maven-enforcer-plugin
as of now am using:
<plugin>
                        <groupId>org.apache.maven.plugins</groupId>
                        <artifactId>maven-enforcer-plugin</artifactId>
                        <dependencies>
                            <dependency>
                                <groupId>com.trivago.rta</groupId>
                                <artifactId>your-custom-maven-fail-rule</artifactId>
                                <version>0.1.6</version>
                            </dependency>
                        </dependencies>
                        <executions>
                            <execution>
                                <phase>post-integration-test</phase>
                                <goals>
                                    <goal>enforce</goal>
                                </goals>
                                <configuration>
                                    <rules>
                                        <yourRule implementation="com.example.YourFailRule" />
                                    </rules>
                                    <fail>true</fail>
                                </configuration>
                            </execution>
                        </executions>
                    </plugin>

**Comment by laxersaz** [comment]
Hi!

Since out custom fail rule is not open source (and does not belong to the Cucable plugin), I cannot disclose this here unfortunatelly.
If you want to write a rule of your own, have a look at the Maven Enforcer documentation: https://maven.apache.org/enforcer/maven-enforcer-plugin/ - especially the section on writing rules here: https://maven.apache.org/enforcer/enforcer-api/writing-a-custom-rule.html

In a nutshell, if you run the tests via failsafe, you will find a failsafe summary report xml file in your maven target after the test run. So your rule could just parse this file, check if there were any errors and fail the build in this case.

**Comment by jdoriyastpl** [comment]
@laxersaz  Thanks for suggestion, I am closing the issue since am able to proceed with Maven Enforcer custom rule.
